app/views/categories/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<section>
+  <%= render partial: 'shared/navbar', locals: { title: @category.name } %>
+
+  <div style='margin-top: 30px;' class='px-5 d-flex flex-row justify-content-between align-items-center p-2 border bg-white no-scroll'>
+    <p class='margin-zero fs-4 fw-bold'>Total Expenses: </p>
+    <p class='margin-zero fs-4'>$ <%= number_with_precision(@total_expenses, precision: 2) %></p>
+  </div>
+
+  <div class="cat-expense-container d-flex flex-column text-center p-1">
+  <br>
+    <div class='expense-container d-flex flex-column container'>
+    <% if @expenses.empty? %>
+      <p class="text-center mx-1">No expenses found for this category</p>
+    <%else%>
+      <% @expenses.each do |expense| %>
+        <div class='expense-info justify-content-center rounded p-3 mb-2 bg-white'>
+          <div class='expense-detail'>
+            <p>Name: <span class='fw-semibold'><%= expense.name%></span></p>
+            <p>Amount: <span class='fw-bold'>$<%= number_with_precision(expense.amount, precision: 2)%></span></p>
+          </div>
+          <%# <div> %>
+          <%# <%= button_to 'X', category_expense_path(expense), method: :delete, data: { turbo: false }, class: 'expense-del-btn' %>
+          <%# </div> %>
+        </div>
+      <% end %>
+    <%end%>
+    
+    </div>
+
+    <div class='d-flex flex-column text-center w-100 p-3'>
+      <%= link_to 'Add New Expense', new_category_expense_path(@category), class:'add-exp-btn text-decoration-none p-2 fs-5 fw-semibold' %>
```

Comment:
- [ ] As per the given design layout, please move your button to the bottom and style it as the 'scan' button in the given design. Also, please give the right font-weight to each element to match the given design. (exp: 'Total expenses')

yours|given
---|---
![image](https://github.com/jonas-45/budget-app/assets/91486791/32fc46db-5d51-4579-879f-1e2ea2de08a4)|![image](https://github.com/jonas-45/budget-app/assets/91486791/57397560-cd26-4dce-bab9-75b04d9a0a51)